modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -9,17 +9,19 @@ The {product-title} cluster log forwarding feature uses a combination of _output
 
 [NOTE]
 ====
-If you want to use only the default internal {product-title} Elasticsearch instance, do not configure any outputs and pipelines. 
+If you want to use only the default internal {product-title} Elasticsearch logstore, do not configure any outputs and pipelines. 
 ==== 
 
-An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+By default, the Cluster Logging Operator creates `logforwarding` configuration file that forwards logs to the default internal Elasticsearch logstore. To use the log forwarding feature, you create a custom `logforwarding` configuration file to send logs to endpoints you specify. In this configuration file, the `disableDefaultForwarding=true` parameter disables the default forwarding behavior. This change assumes you define outputs and pipelines to forward logs for storage. 
```

Comment:
"By default, the Cluster Logging Operator creates `logforwarding` configuration file that forwards logs to the default internal Elasticsearch logstore. ".  That confuse me. 
Could we use 'If elasticsearch are defined in the clusterlogging CRD. The Cluster Logging Operator will enable the fluentd to ship logs the default internal Elasticsearch logstore'

kind: "ClusterLogging"
metadata:
  name: "instance"
  namespace: openshift-logging
spec:
  managementState: "Managed"
  logStore:
    type: "elasticsearch"
    elasticsearch:
      nodeCount: 3
      resources:
        limits:
          memory: 2Gi
        requests:
          cpu: 200m
          memory: 2Gi
      storage:
        size: "20G"
        storageClassName: "gp2"
      redundancyPolicy: "SingleRedundancy"
  visualization:
    type: "kibana"
    kibana:
      replicas: 1
  curation:
    type: "curator"
    curator:
      schedule: "30 3 * * *"
  collection:
    logs:
      type: "fluentd"
      fluentd: {}

Reply:
There is a plugin named forward. How about replace some the word 'forward' with 'sent , ship, copy or transfer'?

Reply:
_By default, the Cluster Logging Operator sends logs to the default internal Elasticsearch logstore, defined in the Cluster Logging Custom Resource. To use the log forwarding feature, you create a custom `logforwarding` configuration file to send logs to endpoints you specify._